content/chapter-03/ApplicationMetrics.md

```diff
@@ -9,13 +9,13 @@
   * Availability percentage, e.g.: Percentage of uptime for interval of time
   * Concurrent users, e.g.: Concurrent user count distribution over a day
   * User satisfaction using [Apdex score](https://www.apdex.org/)
-  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per using of time.
+  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per unit of time.
   
   
-* System level metrics ([Resource]())
-  * CPU utilization, e.g.: CPU utilization percentage per second
-  * Memory utilization, e.g.: Peak memory usage, Percentage of memory in used per application
-  * Network utilization, e.g.: Network bandwith usage per second
+* System level metrics ([Resource](https://blog.heroix.com/blog/how-to-monitor-server-performance))
+  * CPU utilization, e.g.: CPU utilization percentage
+  * Memory utilization, e.g.: Peak memory usage, Percentage of memory in use per application
+  * [Network utilization](https://en.wikibooks.org/wiki/Transwiki:Measuring_network_throughput), e.g.: Bytes send/received per second
   * Disk utilization, e.g.: Disk Read/Writes per second
   * Network outages, e.g.: Percentage of time a network resource is unavailable
```

Comment:
Network outages: percentage of time a network resource is unavailable

Reply:
Done